app/controllers/items_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ItemsController < ApplicationController
   def index
-    @items = Item.all
+    @items = Item.order(:title).page params[:page]
```

Comment:
Don't omit parentheses when you're chaining methods - you're combining two different styles here in the same line, so it might be better to be more consistent and just use parens after `page`